db/migrate/20170918125918_ids_to_bigints.rb

```diff
@@ -68,21 +68,52 @@ class IdsToBigints < ActiveRecord::Migration[5.1]
   ]
   INCLUDED_COLUMNS << [:deprecated_preview_cards, :id] if table_exists?(:deprecated_preview_cards)
```

Comment:
deprecated_preview_cards is *dead*, even if it exists. No need to update it. It's not used in the code. For anything.

Reply:
Sure, I'd be fine removing this line; I just added it in to keep up parity with #5043. (Is there a reason that table hasn't just been dropped yet?)

Reply:
It has been dropped! It's not in the schema. See rake task mastodon:maintenance:remove_deprecated_preview_cards

Reply:
Ah, I suppose my confusion has come from it being created in the migrations and then not removed by them. (For consistency's sake, I often do `rake db:create && rake db:migrate` to make sure I'm not getting something weird from my own schema.rb.) I do see it being ignored by SchemaDumper, though.